tests/integration/sunrisa_integration.py

```diff
@@ -195,25 +200,34 @@ def start_grows_for_shelves_succeeded(message) -> None:
         assert message["succeeded"]
         flag.append(True)
 
-    @sio.on("set_lights_for_grow")
+    @sio.on("set_lights_for_grow")#make sure you test for everything that's printed out in response is present ("power, blue, red, shelves")
     def set_lights_for_grow(message) -> None:
-        assert "grow" in message
         assert "power_level" in message
         assert "red_level" in message
         assert "blue_level" in message
-        flag.append(True)
+        assert "shelves" in message
+        assert message["power_level"] == 9
+        assert message["blue_level"] == 7
+        assert message["red_level"] == 8
+        assert message["shelves"][0]["room_id"] == 1
+        assert message["shelves"][0]["rack_id"] == 2
+        assert message["shelves"][0]["shelf_id"] == 1
```

Comment:
nit: Can we just use variables `room_id`, `rack_id`, and `shelf_id` here? They're all in scope.